docs/source/developing_dallinger_setup_guide.rst

```diff
@@ -680,6 +680,21 @@ Test that your installation works by running:
     dallinger --version
 
 
+Install the Git pre-commit hook
```

Comment:
@vlall Here are the instructions on installing the pre-commit hook. Once installed, it should run `black` on commit.

Reply:
The instructions seem fine, do I need to do anything differently when making a commit to get the contents of my commit to format? It should just auto-format my usual commit, right?